### DISCUSSION TITLE
Calculate horizontal plane and visualize for TurbOPark model

### DISCUSSION THREAD

**Discussion by Edudiro** [NatLabRockies/floris#742] [Q&A]
Hello, I'd like to ask if there are plans to implement `fi.calculate_horizontal_plane() ` for TurbOPark model.

I know it should be possible with the Coarse turbine scan method but my pc crashes for a windfarm with 72 turbines...

If there is a workaround to plot the velocity field I would also appreciate a heads-up.

Thank you!!
Eduardo

**Comment by rafmudaf** [comment]
We do not currently have it in the plans to support the [FlowFieldPlanarGrid](https://github.com/NREL/floris/blob/main/floris/simulation/grid.py#L534) for the TurbOPark model. For reference, the TurbOPark model assumes that any points calculated are turbines and not simply points in the flow field. See https://github.com/NREL/floris/pull/396 and https://github.com/NREL/floris/pull/439.

That being said, @Edudiro I'm happy to collaborate with you if you're interested to explore whether the implementation of this model could be adjusted to support visualization.

**Reply by Edudiro** [reply]
Thanks for your anwer @rafmudaf.

I am currently in the process of understanding the modeling practices of floris and, in particular, the implementation differences between TurbOPark and simpler models like Jensen; just to grasp why [FlowFieldPlanarGrid](https://github.com/NREL/floris/blob/main/floris/simulation/grid.py#L534) is not supported.

Once I get this, I'll be happy to explore TurbOPark visualization.
